hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -78,6 +78,7 @@ function ozonecmd_case
   # This parameter significantly reduces GC pressure for Datanode.
   # Corresponding Ratis issue https://issues.apache.org/jira/browse/RATIS-534.
   RATIS_OPTS="-Dorg.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads=false ${RATIS_OPTS}"
+  OZONE_OPTS="-Dorg.apache.ratis.thirdparty.io.netty.native.workdir=${OZONE_HOME}/temp ${OZONE_OPTS}"
```

Comment:
@szetszwo does this look safe to you?

Reply:
According to https://netty.io/4.0/xref/io/netty/util/internal/NativeLibraryLoader.html , the change here is to use Ozone temp dir instead of the perform temp dir.  So, it seems okay.

@arp7 , do you see any problems which may occur?